thirty-five-mm/app/controllers/application_controller.rb

```diff
@@ -1,5 +1,13 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :current_user, :logged_in?
+
+  def current_user
+    @current_user ||= User.find_by_id(session[:user])
```

Comment:
find_by_xxx methods are deprecated these days. Better to do `find_by(id: session[:user])` here.